lib/skate_web/templates/layout/test_groups.html.eex

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+    <%
+    {static_href_module, static_href_fn} = Application.get_env(:skate, :static_href)
+    static_href = fn conn, path -> apply(static_href_module, static_href_fn, [conn, path]) end
+    %>
+    <head>
+        <meta charset="utf-8"/>
+
+        <title>Skate Test Groups</title>
+
+        <link rel="apple-touch-icon" href="<%= static_href.(@conn, "/images/mbta-logo-t-180.png") %>" type="image/png">
+        <link rel="icon" href="<%= static_href.(@conn, "/images/mbta-logo-t-favicon.png") %>" sizes="32x32" type="image/png">
+        <link rel="icon" href="<%= static_href.(@conn, "/favicon.ico") %>" sizes="16x16" type="image/vnd.microsoft.icon">
+    </head>
```

Comment:
question (non-blocking): I don't see any logos when I run locally, is there something I'm missing about what this is doing?

Reply:
Hmm, I'm seeing the logo when I run it locally - the main spot you should see it is just the favicon in the browser tab.

Reply:
Ah okay yep I see it!